### DISCUSSION TITLE
Bootstrap Doc's Code

### DISCUSSION THREAD

**Discussion by InsiteFX** [twbs/bootstrap#40549] [Q&A]
Is it ok to use code from the Bootstrap Doc's like the Theme Switcher etc;

doc.css & color-modes.js

As long as the Copyright is left intact?

**Comment by julien-deramond** [comment]
Yes, sure, I've added some references associated with the files where you can find more precise legal information if needed:
- [`docs.css`](https://getbootstrap.com/docs/5.3/assets/css/docs.css) is https://creativecommons.org/licenses/by/3.0/deed.en

```
/*!
 * Bootstrap Docs (https://getbootstrap.com/)
 * Copyright 2011-2024 The Bootstrap Authors
 * Licensed under the Creative Commons Attribution 3.0 Unported License.
 * For details, see https://creativecommons.org/licenses/by/3.0/.
 */
```

- [`color-modes.js`](https://getbootstrap.com/docs/5.3/assets/js/color-modes.js) is also https://creativecommons.org/licenses/by/3.0/deed.en

```
/*!
 * Color mode toggler for Bootstrap's docs (https://getbootstrap.com/)
 * Copyright 2011-2024 The Bootstrap Authors
 * Licensed under the Creative Commons Attribution 3.0 Unported License.
 */
```
